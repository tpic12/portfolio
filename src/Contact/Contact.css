.contact-section {
  width: 100%;
  height: 95vh;
  background-color: #ededed;
  display: flex;
  /* margin-top: 100px; */
  justify-content: center;
  align-items: flex-end;
}

.contact-div {
  background-color: rgb(62, 72, 117);
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  height: 80vh;
  width: 100vw;
  border-radius: 200px 200px 0 0;
  color: white;
  font-size: 24px;
}

.contact-div img {
  width: 180px;
  border: 2px solid #ededed;
  border-radius: 5px;
  margin: 8px;
}

.contact-div a {
  text-decoration: none;
  color: white;
}

.contact-div a:hover {
  font-weight: 600;
  font-size: 30px;
  color: rgb(189, 89, 52);
}

.contact-pic-text {
  display: flex;
  flex-direction: row;
}

.contact-list {
  list-style-type: none;
  padding: 0;
  display: flex;
  justify-content: space-evenly;
  align-items: center;
  width: 70vw;
  position: relative;
  top: 130px;
}

.contact-list a {
  color: white;
  font-size: 40px;
}

.contact-list a:hover {
  color: rgb(189, 89, 52);
  font-size: 50px;
}

.contact-list a:active {
  color: white;
}

.contact-text {
  text-align: center;
}

@media screen and (min-width: 720px) {
  .contact-section {
    align-items: center;
  }

  .contact-div {
    border-radius: 0;
  }

  .contact-pic-text {
    width: 80vw;
    max-width: 660px;
    justify-content: space-between;
  }

  .contact-div img {
    width: 250px;
  }

  .contact-text a {
    align-self: center;
  }
}