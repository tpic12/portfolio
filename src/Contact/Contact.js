import React, { Component } from "react";
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import Pic from "../Images/headshot.png";
import "./Contact.css";

class Contact extends Component {
  render() {
    return (
      <section id="contact-section" className="contact-section snap">
        <div className="contact-div">
          <div className="contact-pic-text">
            {/* <img src={Pic} alt="Taylor" /> */}
            <div className="contact-text">
              <h3>
                <span className="thin">It's dangerous to go alone! </span>
                <br />
                <span className="bold">Take this.</span>
              </h3>
              <a
                href="https://taylorpiccarretoportfolio.s3.us-east-2.amazonaws.com/TaylorPiccarretoResume.pdf"
                target="_blank"
                rel="noopener noreferrer"
              >
                Resume
              </a>
            </div>
          </div>
          <ul className="contact-list">
            <li>
              <a
                href="https://www.linkedin.com/in/taylor-piccarreto/"
                target="_blank"
                rel="noopener noreferrer"
              >
                <FontAwesomeIcon icon={["fab", "linkedin"]} />
              </a>
            </li>
            <li>
              <a href="mailto:tpic12@gmail.com">
                <FontAwesomeIcon icon="at" />
              </a>
            </li>
            <li>
              <a
                href="https://github.com/tpic12"
                target="_blank"
                rel="noopener noreferrer"
              >
                <FontAwesomeIcon icon={["fab", "github"]} />
              </a>
            </li>
          </ul>
        </div>
      </section>
    );
  }
}

export default Contact;
