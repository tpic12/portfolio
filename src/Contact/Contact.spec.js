import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Contact from "./Contact";

describe("<Landing />", () => {
  it("renders by default", () => {
    const wrapper = shallow(<Contact />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
