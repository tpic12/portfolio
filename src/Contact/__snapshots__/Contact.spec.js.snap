// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Landing /> renders by default 1`] = `
<section
  className="contact-section snap"
  id="contact-section"
>
  <div
    className="contact-div"
  >
    <div
      className="contact-text"
    >
      <h3>
        <span
          className="thin"
        >
          It's dangerous to go alone! 
        </span>
        <br />
        <span
          className="bold"
        >
          Take this.
        </span>
      </h3>
      <a
        href="https://taylorpiccarretoportfolio.s3.us-east-2.amazonaws.com/TaylorPiccarretoResume.pdf"
        rel="noopener noreferrer"
        target="_blank"
      >
        Resume
      </a>
    </div>
    <ul
      className="contact-list"
    >
      <li>
        <a
          href="https://www.linkedin.com/in/taylor-piccarreto/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <FontAwesomeIcon
            border={false}
            className=""
            fixedWidth={false}
            flip={null}
            icon={
              Array [
                "fab",
                "linkedin",
              ]
            }
            inverse={false}
            listItem={false}
            mask={null}
            pull={null}
            pulse={false}
            rotation={null}
            size={null}
            spin={false}
            swapOpacity={false}
            symbol={false}
            title=""
            transform={null}
          />
        </a>
      </li>
      <li>
        <a
          href="mailto:tpic12@gmail.com"
        >
          <FontAwesomeIcon
            border={false}
            className=""
            fixedWidth={false}
            flip={null}
            icon="at"
            inverse={false}
            listItem={false}
            mask={null}
            pull={null}
            pulse={false}
            rotation={null}
            size={null}
            spin={false}
            swapOpacity={false}
            symbol={false}
            title=""
            transform={null}
          />
        </a>
      </li>
      <li>
        <a
          href="https://github.com/tpic12"
          rel="noopener noreferrer"
          target="_blank"
        >
          <FontAwesomeIcon
            border={false}
            className=""
            fixedWidth={false}
            flip={null}
            icon={
              Array [
                "fab",
                "github",
              ]
            }
            inverse={false}
            listItem={false}
            mask={null}
            pull={null}
            pulse={false}
            rotation={null}
            size={null}
            spin={false}
            swapOpacity={false}
            symbol={false}
            title=""
            transform={null}
          />
        </a>
      </li>
    </ul>
  </div>
</section>
`;
